uaclient/timer/update_contract_info.py

```diff
@@ -28,3 +29,31 @@ def update_contract_info(cfg: UAConfig) -> bool:
             )
             return False
     return True
+
+
+def validate_release_series(cfg: UAConfig, show_message=False):
+    machine_token_file = machine_token.get_machine_token_file(cfg)
+    current_series = system.get_release_info().series
+    only_series = (
+        machine_token_file.entitlements()
+        .get("support", {})
+        .get("entitlement", {})
+        .get("affordances", {})
+        .get("onlySeries", None)
+    )
+    if only_series:
+        allowed_release = system.get_distro_info(only_series)
+        if only_series != current_series:
+            detach()
+            if show_message:
+                print(
+                    messages.PRO_ONLY_ALLOWED_FOR_RELEASE.format(
+                        allowed_release.release,
+                        allowed_release.series_codename,
+                    )
+                )
+            LOG.warning(
+                "Detaching Ubuntu Pro. Previously attached subscription was only valid for Ubuntu %s (%s) release.",  # noqa E501
+                allowed_release.release,
+                allowed_release.series_codename,
+            )
```

Comment:
I usually would not mind that much but let's kill a noqa
```suggestion
            message = messages.PRO_ONLY_ALLOWED_FOR_RELEASE.format(
                allowed_release.release,
                allowed_release.series_codename,
            )
            if show_message:
                print(message)
            LOG.warning(message)
```